wandb/sdk/wandb_run.py

```diff
@@ -1070,8 +1070,16 @@ def _visualization_hack(self, row: Dict[str, Any]) -> Dict[str, Any]:
 
         return row
 
+    def _ignore_step(self) -> bool:
```

Comment:
will replace this with `_ignore_user_step` attribute once we merge this PR:
https://github.com/wandb/client/pull/3308
as part of the attach we will update the argument to `True`...